scipy/special/tests/test_erfinv.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from numpy.testing import assert_allclose, assert_equal
+import pytest
+
+import scipy.special as sc
+
+class TestInverseErrorFunction:
```

Comment:
Nice tests. There are also a couple `erfinv(c)` tests in `test_basic.py`, could you move those here too?

Reply:
Done!